controller/rest/repository.go

```diff
@@ -316,3 +321,45 @@ func handlerScanRepositorySubmit(w http.ResponseWriter, r *http.Request, ps http
 
 	restRespSuccess(w, r, nil, acc, login, &data, "Summit repository scan result")
 }
+
+func getScanReqRootsOfTrust() (scanReqRootsOfTrust []*share.SigstoreRootOfTrust, err error) {
+	clusRootsOfTrust, err := clusHelper.GetAllSigstoreRootsOfTrust()
+	if err != nil {
+		return nil, fmt.Errorf("could not retrieve roots of trust from kv store: %s", err.Error())
+	}
+
+	for key, clusRoot := range clusRootsOfTrust {
+		scanRoot := &share.SigstoreRootOfTrust{
+			Name:           clusRoot.Name,
+			RekorPublicKey: clusRoot.RekorPublicKey,
+			RootCert:       clusRoot.RootCert,
+			SCTPublicKey:   clusRoot.SCTPublicKey,
+		}
+
+		verifiers, err := clusHelper.GetAllSigstoreVerifiersForRoot(key)
+		if err != nil {
+			return scanReqRootsOfTrust, fmt.Errorf("could not retrieve verifiers for root \"%s\": %s", key, err.Error())
+		}
+
+		for _, verifier := range verifiers {
+			scanVerifier := &share.SigstoreVerifier{
```

Comment:
where is share.SigstoreVerifier defined?

Reply:
scanner_service.proto

This is just building the request to the rpc service.